packages/db/schemas/reactions.ts

```diff
@@ -0,0 +1,31 @@
+/* import { relations } from "drizzle-orm"; */
+import { integer, pgTable, primaryKey, text, timestamp } from "drizzle-orm/pg-core";
+
+import { users } from "./users";
+
+export const reactions = pgTable(
+  "reaction",
+  {
+    happeningId: text("happening_id").notNull(),
```

Comment:
Kunne man ha kalt det noe annet? Man skal vel kunne reagere til mer enn bare happenings?

`reactedToKey`? Så kan man lage nøkler for hva som har blitt reacta til.

F.eks et innlegg blir `post:{slug}` og en happening blir `happening:{id eller slug}`